sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/index.marko

```diff
@@ -0,0 +1,129 @@
+import { getAsArray } from "@parameter1/base-cms-object-path";
+import queryFragment from "@parameter1/base-cms-marko-web-theme-monorail/graphql/fragments/content-list"
+import sectionWithChilderenFragment from "../../../../graphql/fragments/section-with-children";
+import config from "./config/top-women-all.js";
+
+$ const { id, alias, name, pageNode } = input;
+$ const blockName = "top-women-page";
+$ const cardBlock = "top-women-page-card";
+
+$ const rootAlias = "awards-honors/top-women";
+$ const currentYear = alias === rootAlias ? Object.keys(config)[Object.keys(config).length - 1] : alias.split("/").pop()
+$ const currentYearData = config[currentYear];
+$ const title = currentYearData ? currentYearData.title : title;
+$ const subtitle = currentYearData ? currentYearData.subtitle : title;
+$ const createdBy = currentYearData ? currentYearData.createdBy : title;
```

Comment:
title in this context would be the one declared on line 13, which to me would indicate if currentYearData doesn't exist therefore title wouldn't either, either default all of these to something aside from title, ideally using the ```defaultValue``` utility function.